lambda-alb/src/AWS/Lambda/ALB.hs

```diff
@@ -9,40 +9,39 @@ where
 import AWS.Lambda.Runtime
 import AWS.Lambda.Runtime.Prelude
 import Data.ByteString (ByteString)
-import Data.CaseInsensitive (CI)
 import Data.HashMap.Strict (HashMap)
 
-import qualified Data.Aeson           as JSON
-import qualified Data.Aeson.Types     as JSON
-import qualified Data.CaseInsensitive as CI
-import qualified Data.HashMap.Strict  as HashMap
-import qualified Network.HTTP.Types   as HTTP
+import qualified Data.Aeson            as JSON
+import qualified Data.Aeson.Types      as JSON
+import qualified Data.CaseInsensitive  as CI
+import qualified Data.HashMap.Strict   as HashMap
+import qualified Network.HTTP.Types    as HTTP
```

Comment:
__suggestion (nitpick):__ It looks like unnecessary whitespace was added here.

Reply:
Let me fix it up. Wish we had a linter for it but will try to build muscle memory.